internal/cmd/internal/olmv1/catalog_installed_get.go

```diff
@@ -0,0 +1,38 @@
+package olmv1
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/operator-framework/kubectl-operator/internal/cmd/internal/log"
+	v1action "github.com/operator-framework/kubectl-operator/internal/pkg/v1/action"
+	"github.com/operator-framework/kubectl-operator/pkg/action"
+)
+
+// NewCatalogInstalledGetCmd handles get commands in the form of:
+// catalog(s) [operatorName] - this will either list all the installed operators
+// if no operatorName has been provided or display the details of the specific
+// one otherwise
+func NewCatalogInstalledGetCmd(cfg *action.Configuration) *cobra.Command {
+	i := v1action.NewCatalogInstalledGet(cfg)
+	i.Logf = log.Printf
+
+	cmd := &cobra.Command{
+		Use:     "catalog",
```

Comment:
I don't know if we're consistent with this suggestion now, but it would be nice to include the optional arg in the `Use` string. E.g. something like:

```suggestion
		Use:     "catalog [catalog_name]",
```

If we're not already consistently doing this, capturing a follow-up issue to improve our usage text would suffice.

Reply:
thanks, we were actually doing that consistently both in our old alpha v1 commands as well as across olmv0 commands in general and I extended the usage to match